### PR TITLE
Provide project-level badges in the README

### PR DESCRIPTION
**Pull request by AnthonyMastrean**
For builds, chat, and gems!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -84,3 +89,10 @@ Targets:
 * pacman (ArchLinux) packages
 
 .. include: docs/contributing
+
+.. |Build| image:: https://img.shields.io/travis/jordansissel/fpm.svg
+   :target: https://travis-ci.org/jordansissel/fpm
+.. |Chat| image:: https://img.shields.io/gitter/room/jordansissel/fpm.svg
```
Comment (jordansissel) [inline README.rst]: http://fpm.readthedocs.io/en/latest/contributing.html recommends #fpm on freenode irc. If you desire folks relocate to gitter, then some investment needs to occur in moving folks from IRC to Gitter.

**Comment by AnthonyMastrean** [comment]
Oops, that was my mistake! Let me see if there’s an IRC badge :)

**Comment by jordansissel** [comment]
@AnthonyMastrean no worries, review makes for good teamwork to catch things :)

Thanks for helping this project!

**Comment by AnthonyMastrean** [comment]
README badges might help draw attention and effort to fixing broken builds! (See #1437)

**Comment by jordansissel** [comment]
Looks good.

**Pull request merged**